OpenEdXMobile/src/main/java/org/edx/mobile/notifications/services/NotificationService.java

```diff
@@ -0,0 +1,94 @@
+package org.edx.mobile.notifications.services;
+
+import android.app.NotificationChannel;
+import android.app.NotificationManager;
+import android.app.PendingIntent;
+import android.content.Context;
+import android.content.Intent;
+import android.media.RingtoneManager;
+import android.net.Uri;
+import android.os.Build;
+import android.support.v4.app.NotificationCompat;
+
+import com.google.firebase.messaging.FirebaseMessagingService;
+import com.google.firebase.messaging.RemoteMessage;
+
+import org.edx.mobile.R;
+import org.edx.mobile.base.MainApplication;
+import org.edx.mobile.core.IEdxEnvironment;
+import org.edx.mobile.logger.Logger;
+import org.edx.mobile.view.SplashActivity;
+
+
+public class NotificationService extends FirebaseMessagingService {
+    public static final String NOTIFICATION_TOPIC_RELEASE = "edx_release_notification_android";
+    public static final String NOTIFICATION_RELEASE_CHANNEL = "edx_notification_release_channel";
+    public static final String NOTIFICATION_RELEASE_CHANNEL_TITLE = "Release";
```

Comment:
It should go in strings.xml if this content will appear to user, otherwise NOTIFICATION_RELEASE_CHANNEL_TITLE = "NOTIFICATION_RELEASE_CHANNEL_TITLE" looks more good to me.